src/Features/CSharp/Portable/FullyQualify/CSharpFullyQualifyCodeFixProvider.cs

```diff
@@ -53,49 +48,5 @@ public CSharpFullyQualifyCodeFixProvider()
 
         public override ImmutableArray<string> FixableDiagnosticIds { get; } =
             ImmutableArray.Create(CS0103, CS0104, CS0246, CS0305, CS0308, IDEDiagnosticIds.UnboundIdentifierId);
-
-        protected override bool CanFullyQualify(Diagnostic diagnostic, SyntaxNode node, [NotNullWhen(true)] out SimpleNameSyntax? simpleName)
```

Comment:
all this logic moved, unchnaged into the new CSharpFullyQualifyService lang service.